### PR TITLE
Simple optimization in IsKeyPressed

### PR DESCRIPTION
**Pull request by corfe83**
This is my first change to ebiten. I will start with something small.

I noticed IsKeyPressed is iterating through every possible key to find the one being asked about. This change fixes it to only look at the key being asked about.

Obviously IsKeyPressed calls are not a big deal for performance, but it feels like the right thing to do.

### CODE REVIEW

File: genkeys.go [hunk 1]
```diff
@@ -342,6 +342,11 @@ var glfwKeyCodeToKey = map[glfw.Key]driver.Key{
 {{range $index, $name := .DriverKeyNames}}glfw.Key{{$name}}: driver.Key{{$name}},
 {{end}}
 }
+
+var keyCodeToglfwKey = map[driver.Key]glfw.Key{
```
Comment (hajimehoshi) [inline genkeys.go]: `keyCodeToGLFWKey` sounds better

Comment (corfe83) [reply genkeys.go]: Addressed in latest push

**Comment by corfe83** [comment]
> LGTM
> 
> Please update this PR by merging or rebasing, then I'll merge this PR.

Wow, quick response! Just rebased and force pushed this branch.

**Comment by corfe83** [comment]
> > LGTM
> > Please update this PR by merging or rebasing, then I'll merge this PR.
> 
> Wow, quick response! Just rebased and force pushed this branch.

Actually, something is off. Fixing.

**Comment by corfe83** [comment]
> > > LGTM
> > > Please update this PR by merging or rebasing, then I'll merge this PR.
> > 
> > 
> > Wow, quick response! Just rebased and force pushed this branch.
> 
> Actually, something is off. Fixing.

Fixed now.

**Pull request merged**